Documentation/Legacy/Index.rst

```diff
@@ -59,19 +59,24 @@ i.e. if you use your own update mechanism.
 
 This feature is already disabled when TYPO3 is installed via Composer.
 
-To disable the core updater, you can set this environment variable::
+To disable the Core updater, you can set this environment variable:
+
+.. code-block:: none
+   :caption: Environment variable
 
    TYPO3_DISABLE_CORE_UPDATER=1
 
 For example in Apache:
 
 .. code-block:: apacheconf
+   :caption: /etc/httpd/conf/httpd.conf
```

Comment:
```suggestion
   :caption: .htaccess
```

It is more likely that people set this in their `.htaccess`. This file is more likely available on shared hostings than the global server config.